common/src/main/java/org/jboss/hal/testsuite/page/config/elytron/FactoryPage.java

```diff
@@ -0,0 +1,53 @@
+package org.jboss.hal.testsuite.page.config.elytron;
+
+import static org.jboss.hal.testsuite.page.config.elytron.ElytronPageConstants.ELYTRON_SUBSYTEM_LABEL;
+
+import org.jboss.arquillian.graphene.Graphene;
+import org.jboss.arquillian.graphene.findby.ByJQuery;
+import org.jboss.hal.testsuite.fragment.ConfigFragment;
+import org.jboss.hal.testsuite.fragment.shared.util.ResourceManager;
+import org.jboss.hal.testsuite.page.ConfigPage;
+import org.jboss.hal.testsuite.util.PropUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class FactoryPage extends ConfigPage {
+
+    public FactoryPage navigate2application() {
```

Comment:
Won't placing a digit to method's name cause a confusion? In test cases digits are used as indexes...